frontend/packages/kubevirt-plugin/src/components/create-vm/forms/boot-source-form.tsx

```diff
@@ -169,6 +169,9 @@ const AdvancedSection: React.FC<AdvancedSectionProps> = ({
                 provisioner={provisioner}
                 loaded
                 availableAccessModes={initialAccessModes}
+                accMode={
+                  isSPSettingProvided && !isSCNameEmpty ? spAccessMode?.getValue() : undefined
```

Comment:
If I'm reading this correctly, `isSPDefaults` is true here, which means we already know that the conditional expression here is true. Am I missing something?

Reply:
actually `isSPSettingProvided && !isSCNameEmpty` is not a trivial check to do, in line 159 there's a `:` sign that I assume you've missed which basically say that `isSPDefaults` is false so I'm checking if there are SP settings to give as the default values to the radio copmonents selection.